bin/fluentd

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env ruby
 # -*- coding: utf-8 -*-
 
-RubyVM::InstructionSequence.compile_option = {trace_instruction: false} rescue nil
+if defined?(RubyVM::InstructionSequence)
+  RubyVM::InstructionSequence.compile_option = {trace_instruction: false}
```

Comment:
Could you add an explanation of which platform is available for this kind of optimization?

Reply:
If an explanation is added, it looks ok for me.

Reply:
I added in https://github.com/fluent/fluentd/pull/3376/commits/2404104e305e68fb568d0f52f4e1745130bb922a.